OracleDatabase/RAC/OracleDNSServer/dockerfiles/latest/runOracle.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2021 Oracle and/or its affiliates. All rights reserved.
```

Comment:
The date format is incorrect as it should be "1982, 2021" but I doubt this script was created in 1982. The copyright statement should cover the year the current file was first created and the year in which it was last modified. Also, we no longer add "All rights reserved." so that should be removed from all the copyright statements